[2021-08-13 08:44:10,343] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: pipeline_try.friday_Task 2021-08-13T08:44:04.353394+00:00 [queued]>
[2021-08-13 08:44:10,381] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: pipeline_try.friday_Task 2021-08-13T08:44:04.353394+00:00 [queued]>
[2021-08-13 08:44:10,382] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-08-13 08:44:10,384] {taskinstance.py:1088} INFO - Starting attempt 1 of 2
[2021-08-13 08:44:10,385] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-08-13 08:44:10,411] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): friday_Task> on 2021-08-13T08:44:04.353394+00:00
[2021-08-13 08:44:10,418] {standard_task_runner.py:52} INFO - Started process 6051 to run task
[2021-08-13 08:44:10,452] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'pipeline_try', 'friday_Task', '2021-08-13T08:44:04.353394+00:00', '--job-id', '146', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_try1.py', '--cfg-path', '/tmp/tmpnpjgzu3r', '--error-file', '/tmp/tmpdb6m4z_k']
[2021-08-13 08:44:10,482] {standard_task_runner.py:77} INFO - Job 146: Subtask friday_Task
[2021-08-13 08:44:10,626] {logging_mixin.py:104} INFO - Running <TaskInstance: pipeline_try.friday_Task 2021-08-13T08:44:04.353394+00:00 [running]> on host 99300daf4d26
[2021-08-13 08:44:10,733] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=ameysonje242000@gmail.com
AIRFLOW_CTX_DAG_OWNER=amey
AIRFLOW_CTX_DAG_ID=pipeline_try
AIRFLOW_CTX_TASK_ID=friday_Task
AIRFLOW_CTX_EXECUTION_DATE=2021-08-13T08:44:04.353394+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-08-13T08:44:04.353394+00:00
[2021-08-13 08:44:10,739] {logging_mixin.py:104} INFO - From friday_Task, so its Friday today
[2021-08-13 08:44:10,739] {python.py:151} INFO - Done. Returned value was: None
[2021-08-13 08:44:10,752] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=pipeline_try, task_id=friday_Task, execution_date=20210813T084404, start_date=20210813T084410, end_date=20210813T084410
[2021-08-13 08:44:10,794] {taskinstance.py:1265} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2021-08-13 08:44:10,816] {local_task_job.py:149} INFO - Task exited with return code 0
